test/scenarios-bootc/releases/el100-lrel@optional.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Sourced from scenario.sh and uses functions defined there.
+
+# Redefine network-related settings to use the dedicated network bridge
+VM_BRIDGE_IP="$(get_vm_bridge_ip "${VM_MULTUS_NETWORK}")"
+# shellcheck disable=SC2034  # used elsewhere
+WEB_SERVER_URL="http://${VM_BRIDGE_IP}:${WEB_SERVER_PORT}"
+
+start_image="rhel100-bootc-brew-${LATEST_RELEASE_TYPE}-with-optional"
+
+scenario_create_vms() {
+    exit_if_image_not_found "${start_image}"
+
+    prepare_kickstart host1 kickstart-bootc.ks.template "${start_image}"
+    # Two nics - one for macvlan, another for ipvlan (they cannot enslave the same interface)
+    launch_vm --boot_blueprint rhel100-bootc --network "${VM_MULTUS_NETWORK},${VM_MULTUS_NETWORK}"
+}
+
+scenario_remove_vms() {
+    exit_if_image_not_found "${start_image}"
+
+    remove_vm host1
+}
+
+scenario_run_tests() {
+    exit_if_image_not_found "${start_image}"
+
+    run_tests host1 \
+        --variable "PROMETHEUS_HOST:$(hostname)" \
+        --variable "PROMETHEUS_PORT:9092" \
+        --variable "LOKI_HOST:$(hostname)" \
+        --variable "LOKI_PORT:3200" \
+        --variable "PROM_EXPORTER_PORT:8889" \
+        suites/optional/
```

Comment:
```suggestion
    run_tests host1 \
        --variable "PROMETHEUS_HOST:$(hostname)" \
        --variable "PROMETHEUS_PORT:9093" \
        --variable "LOKI_HOST:$(hostname)" \
        --variable "LOKI_PORT:3201" \
        --variable "PROM_EXPORTER_PORT:8889" \
        suites/optional/
```

changing the ports to not reuse the same port (it'll cause an error trying to open same port) from https://github.com/openshift/microshift/blob/main/test/scenarios-bootc/releases/el96-lrel%40optional.sh scenario